src/status_im2/contexts/wallet/account/view.cljs

```diff
@@ -63,38 +64,44 @@
    {:id :about :label (i18n/label :t/about) :accessibility-label :about}])
 
 (defn view
-  []
-  (let [top          (safe-area/get-top)
-        selected-tab (reagent/atom (:id (first tabs-data)))]
+  [account-address]
+  (let [top             (safe-area/get-top)
+        selected-tab    (reagent/atom (:id (first tabs-data)))]
     (fn []
-      [rn/view
-       {:style {:flex       1
-                :margin-top top}}
-       [quo/page-nav
-        {:type              :wallet-networks
-         :background        :blur
-         :icon-name         :i/close
-         :on-press          #(rf/dispatch [:navigate-back])
-         :networks          networks-list
-         :networks-on-press #(js/alert "Pressed Networks")
-         :right-side        :account-switcher
-         :account-switcher  {:customization-color :purple
-                             :on-press            #(rf/dispatch [:show-bottom-sheet
-                                                                 {:content             account-options
-                                                                  :gradient-cover?     true
-                                                                  :customization-color :purple}])
-                             :emoji               "🍑"}}]
-       [quo/account-overview temp/account-overview-state]
-       [quo/wallet-graph {:time-frame :empty}]
-       [quo/wallet-ctas
-        {:send-action #(rf/dispatch [:open-modal :wallet-select-address])
-         :buy-action  #(rf/dispatch [:show-bottom-sheet
-                                     {:content buy-drawer}])}]
-       [quo/tabs
-        {:style          style/tabs
-         :size           32
-         :default-active @selected-tab
-         :data           tabs-data
-         :on-change      #(reset! selected-tab %)
-         :scrollable?    true}]
-       [tabs/view {:selected-tab @selected-tab}]])))
+      (let [account-address (or account-address (rf/sub [:get-screen-params :wallet-accounts]))
+            account         (rf/sub [:wallet-2/account account-address])]
+        [rn/view
```

Comment:
same here  for top screen's option should be used

Reply:
`:options   {:insets {:top? true}}`

Reply:
This gets added in screens.cljs though, right @flexsurfer ?

Reply:
right

Reply:
Done